src/apps/mean-finance/optimism/mean-finance.optimism-airdrop.contract-position-fetcher.ts

```diff
@@ -0,0 +1,83 @@
+import { Inject } from '@nestjs/common';
+import { BigNumber, BigNumberish, Contract } from 'ethers';
+import Axios from "axios";
+
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { PositionTemplate } from '~app-toolkit/decorators/position-template.decorator';
+import { DefaultDataProps } from '~position/display.interface';
+import { MetaType } from '~position/position.interface';
+import { ContractPositionTemplatePositionFetcher } from '~position/template/contract-position.template.position-fetcher';
+import {
+  DefaultContractPositionDefinition,
+  GetDefinitionsParams,
+  GetDisplayPropsParams,
+  GetTokenBalancesParams,
+  GetTokenDefinitionsParams,
+  UnderlyingTokenDefinition,
+} from '~position/template/contract-position.template.types';
+
+import { MeanFinanceContractFactory, MeanFinanceOptimismAirdrop } from '../contracts';
+
+const TOKEN_DISTRIBUTOR = '0xf453cc2cb0d016a028a34162cf1f9efbb799c2d7';
+const OP_TOKEN = '0x4200000000000000000000000000000000000042';
+
+export interface OptimismAirdropCampaingResponse {
+  positions: {
+    id: number;
+    from: string;
+    to: string;
+    version: 'beta' | 'vulnerable';
+    volume: number;
+  }[];
+  totalBoostedVolume: number;
+  op: string;
+  proof: string[];
+}
+
+@PositionTemplate()
+export class OptimismMeanFinanceOptimismAirdropAirdropContractPositionFetcher extends ContractPositionTemplatePositionFetcher<Contract> {
```

Comment:
`OptimismMeanFinanceOptimismAirdropAirdropContractPositionFetcher` contains duplicate `Airdrop` in the name